fenix/app/src/main/res/values/strings.xml

```diff
@@ -1299,12 +1299,16 @@
     <!-- Subtitle for the tabs item in Delete browsing data, parameter will be replaced with the number of open tabs -->
     <string name="preferences_delete_browsing_data_tabs_subtitle">%d tabs</string>
     <!-- Title for the data and history items in Delete browsing data -->
-    <string name="preferences_delete_browsing_data_browsing_data_title">Browsing history and site data</string>
+    <string name="preferences_delete_browsing_data_browsing_data_title" tools:ignore="UnusedResources">Browsing history and site data</string>
+    <!-- Title for the history item in Delete browsing data -->
+    <string name="preferences_delete_browsing_data_browsing_history_title">Browsing history</string>
     <!-- Subtitle for the data and history items in delete browsing data, parameter will be replaced with the
         number of history items the user has -->
     <string name="preferences_delete_browsing_data_browsing_data_subtitle">%d addresses</string>
     <!-- Title for the cookies item in Delete browsing data -->
-    <string name="preferences_delete_browsing_data_cookies">Cookies</string>
+    <string name="preferences_delete_browsing_data_cookies" tools:ignore="UnusedResources">Cookies</string>
+    <!-- Title for the cookies and site data items in Delete browsing data -->
+    <string name="preferences_delete_browsing_data_cookies_and_site_data">Cookies and site data</string>
```

Comment:
We have a [specific process](https://github.com/mozilla-mobile/firefox-android/blob/main/fenix/docs/Working-with-Strings.md) for handling strings here. Basically, these lines that are being removed should actually be kept and have the added attribute `moz:removedIn="118"`.